libmscore/xml.cpp

```diff
@@ -625,6 +625,36 @@ QString Xml::xmlString(const QString& s)
       }
 
 //---------------------------------------------------------
+//   tagEscape
+//---------------------------------------------------------
+
+QString Xml::tagEscape(QString s, QString tag)
+      {
+      QString openTag = "<" + tag + ">";
+      QString openProxy = "!!" + tag + "!!";
+      QString closeTag = "</" + tag + ">";
+      QString closeProxy = "!!/" + tag + "!!";
+      s.replace(openTag, openProxy);
+      s.replace(closeTag, closeProxy);
+      s = Xml::xmlString(s);
+      s.replace(openProxy, openTag);
+      s.replace(closeProxy, closeTag);
+      return s;
+      }
+
+//---------------------------------------------------------
+//   unEscape
+//---------------------------------------------------------
+
+QString Xml::unEscape(QString s)
```

Comment:
Actually, this doesn't work.  It does a nice job on the conversion fo the entities, but we lose the `<sym>` tags (and the others too.  I guess I could use the same trick of protecting the tags first, but it really seems like there should already be some function to do just convert the entities.